pkg/handler/snapshots.go

```diff
@@ -76,14 +76,13 @@ func (sh *SnapshotHandler) getRepoConfigurationFile(c echo.Context) error {
 	_, orgID := getAccountIdOrgId(c)
 	uuid := c.Param("uuid")
 	snapshotUUID := c.Param("snapshot_uuid")
-	var repoConfigFile string
 
 	err := sh.DaoRegistry.Snapshot.InitializePulpClient(c.Request().Context(), orgID)
 	if err != nil {
 		return ce.NewErrorResponse(ce.HttpCodeForDaoError(err), "Error initializing pulp client", err.Error())
 	}
 
-	repoConfigFile, err = sh.DaoRegistry.Snapshot.GetRepositoryConfigurationFile(orgID, snapshotUUID, uuid)
+	repoConfigFile, err := sh.DaoRegistry.Snapshot.GetRepositoryConfigurationFile(orgID, snapshotUUID, uuid, c.Request().Header.Get("referer"))
```

Comment:
using referer doesn't seem to work when using something like curl (and might not work with when using a direct api call).  I *think* this might work better:  `c.Request().Host`      i think this returns the same host:port that the client requested.  We'll want to try that in ephemeral though